src/plonk/mod.rs

```diff
@@ -77,6 +87,18 @@ pub struct RelaxedPlonkInstance<C: CurveAffine> {
     pub(crate) y: C::ScalarExt,
 }
 
+impl<C: CurveAffine> Default for RelaxedPlonkInstance<C> {
+    fn default() -> Self {
```

Comment:
where the default value is used?

Reply:
We have an empty instance being created at the very beginning of folding

Reply:
got it. Then we need make sure the default value satisfy the plonkish relation